src/RestServer/Binder/UInt160Binder.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// UInt160Binder.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Neo.Plugins.RestServer.Binder
+{
+    internal class UInt160Binder : IModelBinder
```

Comment:
How about the same provider for UInt256?

Reply:
There is no alternative formatting for `UInt256`. Unlike `UInt160` can be `scripthash` or `160HASH` format.